docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md

```diff
@@ -99,7 +99,11 @@ mage -Update c:\HelloWorldDeployment\HelloWorld.deploy -CertFile cert.pfx
 
  Application manifests also support custom trust sections. This helps your application obey the security principle of requesting least permission, as you can configure the manifest to demand only those specific permissions that the application requires in order to execute. *Mage.exe* does not directly support adding a custom trust section. You can add one using a text editor, an XML parser, or the graphical tool *MageUI.exe*. For more information about how to use *MageUI.exe* to add custom trust sections, see [MageUI.exe (Manifest Generation and Editing Tool, Graphical Client)](../../../docs/framework/tools/mageui-exe-manifest-generation-and-editing-tool-graphical-client.md).
 
- New manifests that are created with version 4 of *Mage.exe*, which is included with Visual Studio 2010, target the [!INCLUDE[net_client_v40_long](../../../includes/net-client-v40-long-md.md)]. To target earlier versions of the .NET Framework, you must use an earlier version of *Mage.exe*. When adding or removing assemblies from an existing manifest, or re-signing an existing manifest, *Mage.exe* does not update the manifest to target the [!INCLUDE[net_client_v40_long](../../../includes/net-client-v40-long-md.md)]. The following tables show these features and restrictions.
+Visual Studio 2017 includes version 4.6.1 of *Mage.exe*. Manifests created with this version of *Mage.exe* target .NET Framework 4. To target older versions of the .NET Framework, use an earlier version of *Mage.exe*.
```

Comment:
Please verify this version info. This is what I saw in the test manifest I created:


  <dependency>
    <dependentOS>
      <osVersionInfo>
        <os majorVersion="4" minorVersion="10" buildNumber="0" servicePackMajor="0" />
      </osVersionInfo>
    </dependentOS>
  </dependency>
  <dependency>
    <dependentAssembly dependencyType="preRequisite" allowDelayedBinding="true">
      <assemblyIdentity name="Microsoft.Windows.CommonLanguageRuntime" version="4.0.30319.0" />
    </dependentAssembly>
  </dependency>